app/src/newdocs/content/introduction.mdx

```diff
@@ -1,6 +1,128 @@
+        </Link>
+    </Panel>
+    <Panel >
+        <h5>Tutorials</h5>
+        <p>Follow hands-on tutorials to learn how to use Streamr in practice.</p>
+        <Link className={styles.link} to={formatPath(links.newdocs.tutorials)}>
+            Jump to section
+            <SvgIcon name="back" className={styles.forwardArrow} />
+        </Link>
+    </Panel>
+</div>
+
+<hr />
+
+## What is Streamr?
+
+Streamr is an open source, crowdfunded, decentralized platform for realtime data. At its heart is the <Link to={links.site.network}>Streamr Network</Link>, which transports <Link to={links.newdocs.streams}>streams</Link> of messages from data publishers to subscribers, appearing to the user as a global publish/subscribe messaging service.
+
+It is similar in functionality to message queues, enterprise messaging systems, MQTT brokers, Apache Kafka, or pub/sub cloud messaging services. The key features of such systems, including Streamr, are:
+
+- Publishing messages in order to make them available to subscribers
+- Subscribing to messages in order to process them as they occur
+- Storing messages for later use
+
+You can interact with Streamr using the <Link to={links.newdocs.core}>Core</Link> user interface, <Link to={links.newdocs.api}>SDKs</Link>, or directly via the <Link to={links.newdocs.api}>API</Link>.
+
+In addition to the Network, the Streamr stack includes a <Link to={links.newdocs.marketplace}>Marketplace</Link> for data sharing and monetization, and <Link to={links.newdocs.canvases}>Canvases</Link> for data processing and analytics as part of the Core webapp.
+
+Using the built-in tooling is optional, as Streamr also integrates with many popular data frameworks including <Link to={links.blogPosts.spark}>Apache Spark</Link> and <Link to={links.blogPosts.nodered}>Node-RED</Link>.
+
+To start using Streamr right away, check out the <Link to={links.newdocs.gettingStarted}>Getting Started</Link> section and the <Link to={links.newdocs.tutorials}>Tutorials.</Link>
+
+## Why use Streamr?
+
+The big difference to comparable data frameworks is that Streamr aims to be *decentralized* and *peer-to-peer*. There are plenty of advantages to decentralization:
+
+- You don't need to set up any servers to use the service
+- You don't need to trust any third party with your data
+- You benefit from the network effects of everyone participating in the same network, instead of having to operate in an isolated silo
+
+Instead of a centralized party, the Network is run by its community of users, and it heavily relies on cryptography to remove the need for trust. This creates a permissionless and neutral network for realtime data, which is highly suitable for use cases such as data sharing, data monetisation, and the <Link to={links.external.decentralizedWeb}>decentralised web</Link>, extending the capabilities of the underlying internet without compromising its open and decentralised nature.
```

Comment:
```suggestion
Instead of a centralized party, the Network is run by its community of users, and it heavily relies on cryptography to remove the need for trust. This creates a permissionless and neutral network for realtime data, which is highly suitable for use cases such as data sharing, data monetisation, and the <a target="_blank" href={links.external.decentralizedWeb}>decentralised web</a>, extending the capabilities of the underlying internet without compromising its open and decentralised nature.
```